File: sdk/java/mil/nga/giat/mage/sdk/datastore/user/Permissions.java
package mil.nga.giat.mage.sdk.datastore.user;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Collection;

public class Permissions implements Serializable {

	private static final long serialVersionUID = -1912604919150929355L;

	private Collection<Permission> permissions = new ArrayList<Permission>();

	public Permissions() {

	}

	public Permissions(Collection<Permission> permissions) {
		super();
		this.permissions = permissions;
	}

	public Collection<Permission> getPermissions() {
		return permissions;
	}

	public void setPermissions(Collection<Permission> permissions) {
		this.permissions = permissions;
	}

}

File: mage/src/main/java/mil/nga/giat/mage/sdk/datastore/user/Permissions.kt
package mil.nga.giat.mage.sdk.datastore.user

import java.io.Serializable

class Permissions : Serializable {
   var permissions = emptyList<Permission>()

   constructor()

   constructor(permissions: List<Permission>) : super() {
      this.permissions = permissions
   }

   companion object {
      private const val serialVersionUID = -1912604919150929355L
   }
}